rust/bridge/shared/src/lib.rs

```diff
@@ -5,6 +5,7 @@
 
 #![allow(clippy::missing_safety_doc)]
 #![deny(clippy::unwrap_used)]
+#![feature(min_const_generics)]
```

Comment:
This is in stable in the latest rustc, so it can go away when we bump the compiler requirement.